02_Working_With_ASDF_Files/02-Working_With_ASDF_Files.ipynb

```diff
@@ -0,0 +1,382 @@
+{
```

Comment:
<p>I think there needs to be a comment about where u.Quantity comes from and a one line summary of what it does (e.g., Quantities are scalars or arrays with associated units)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/asdf-format/scipy2022tutorial/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
done